Shades/CloseWindowIconView.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  CloseWindowIconView.swift
+//  Shades
+//
+//  Created by Warren Henning on 5/12/18.
+//  Copyright © 2018 Patrick Thomson. All rights reserved.
+//
+
+import Cocoa
+
+class CloseWindowIconView: NSView {
+    static let width: CGFloat = 24
```

Comment:
Yeah, it should be an `NSRect`.

How do I test taking the backing scale factor into account? Test on a retina display? I thought Cocoa used logical points for images and drawing so that you didn't have to worry about it.

I will make sure it works on my external displays and my laptop's retina display, but I'm not sure what else to do right now.